sasjs-tests/package.json

```diff
@@ -4,7 +4,7 @@
   "homepage": ".",
   "private": true,
   "dependencies": {
-    "@sasjs/adapter": "^2.1.0",
+    "@sasjs/adapter": "file:../build/sasjs-adapter-5.0.0.tgz",
```

Comment:
is this correct?

Reply:
This should ideally point to the latest published version.